### PR TITLE
fix links in index.html for sdk docs

### PR DESCRIPTION
**Pull request by keertip**
@sethladd

### CODE REVIEW

File: lib/src/html_generator.dart [hunk 1]
```diff
@@ -448,7 +448,7 @@ body {
 }
 
 String _getFileNameFor(Library library) {
-  return '${library.name.replaceAll('.', '_')}.html';
+  return '${library.name.replaceAll('.', '_').replaceAll(':', '_')}.html';
```
Comment (sethladd) [inline lib/src/html_generator.dart]: can we add a comment explaining why we are doing this? :)

Comment (keertip) [inline lib/src/html_generator.dart]: done.

**Comment by sethladd** [comment]
Is it easy to write a test for this?

**Comment by keertip** [comment]
Not right now, since it is a private method. Will refactor and add tests in a separate cl.

**Pull request merged**